src/evaluate-by-operator/operator/divide.js

```diff
@@ -6,12 +6,15 @@ export const SYMBOL = '/';
 export default function func(first, ...rest) {
   const result = rest.reduce((acc, value) => acc / toNumber(value), toNumber(first));
 
+  const has_consecutive_zeros = [first, ...rest].some((value, index, array) => value === 0 && array[index + 1] === 0);
```

Comment:
I think this will miss cases like [2,4,0,4,3,0]

Reply:
As discussed with @espringer0 , this is not ever hit with more than 2 operands in practice, so I'm ok moving forward with this